src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -641,9 +638,42 @@
         }
     ],
     "models": {
+        "cpu_profile_result": {
+            "id": "cpu_profile_result",
+            "description": "top level object for a cpu profile request",
+            "properties" : {
+                "schema": {
+                    "description": "the schema version of the response",
+                    "type": "int"
+                },
+                "arch": {
+                    "description": "the CPU architecture the profile was taken on, one of [amd64, arm64]",
+                    "type": "string"
+                },
+                "version": {
+                    "description": "the version of Redpanda the profile was taken on",
+                    "type": "string"
+                },
+                "wait_ms": {
+                    "description": "the requested sample period, in milliseconds, if specified using wait_ms query parameter, or missing otherwise",
```

Comment:
```suggestion
                    "description": "The requested sample period, in milliseconds, if specified using the wait_ms query parameter, or missing otherwise",
```
Does "missing otherwise" mean it's not always included in the body? If so, is it the only field that could be left out? Wondering if it's okay to skip "or missing otherwise" if it's intuitive enough

Reply:
> Does "missing otherwise" mean it's not always included in the body?

That's right, if the sample was taking without a wait_ms (which is a different behavior), the wait_ms attribute will not be present.

> If so, is it the only field that could be left out?

Not 100% sure, but the large majority of fields are always present.

> Wondering if it's okay to skip "or missing otherwise" if it's intuitive enough

I don't think it's exactly intuitive and it seems important to document the behavior? Anyone parsing the field needs to be prepared for it to be missed (which usually doesn't happy "naturally") and would also need to know what the meaning of a missing value is.